src/display/canvas.js

```diff
@@ -1313,11 +1313,20 @@ class CanvasGraphics {
       0,
     ]);
     maskToCanvas = Util.transform(maskToCanvas, [1, 0, 0, 1, 0, -height]);
-    const cord1 = Util.applyTransform([0, 0], maskToCanvas);
-    const cord2 = Util.applyTransform([width, height], maskToCanvas);
-    const rect = Util.normalizeRect([cord1[0], cord1[1], cord2[0], cord2[1]]);
-    const drawnWidth = Math.round(rect[2] - rect[0]) || 1;
-    const drawnHeight = Math.round(rect[3] - rect[1]) || 1;
+    const cords = [
+      [0, 0],
+      [width, height],
+      [0, height],
+      [width, 0],
+    ].map(cord => Util.applyTransform(cord, maskToCanvas));
+    const xList = cords.map(cord => cord[0]);
+    const yList = cords.map(cord => cord[1]);
+    const minX = Math.min(...xList);
+    const minY = Math.min(...yList);
+    const maxX = Math.max(...xList);
+    const maxY = Math.max(...yList);
```

Comment:
Unless I'm missing something, it looks like this code is re-implementing the following helper?
https://github.com/mozilla/pdf.js/blob/5f0e5609499374c01cb9ee534f2d44bbd2ae14fb/src/shared/util.js#L725-L738

Can't this be written as follows instead?
```suggestion
    const [minX, minY, maxX, maxY] = Util.getAxialAlignedBoundingBox(
      [0, 0, width, height],
      maskToCanvas
    );
```